test/spec/util/util.js

```diff
@@ -225,4 +225,31 @@ describe('iD.util', function() {
             expect(iD.utilUnicodeCharsTruncated('😎😬😆😵😴😄🙂🤔', 255)).to.eql('😎😬😆😵😴😄🙂🤔');
         });
     });
+
+    describe('utilDisplayName', function() {
+        it('returns the name if tagged with a name', function() {
+            expect(iD.utilDisplayName({tags: {name: 'East Coast Greenway'}})).to.eql('East Coast Greenway');
+        });
+        it('distinguishes unnamed features by ref', function() {
+            expect(iD.utilDisplayName({tags: {ref: '66'}})).to.eql('66');
```

Comment:
This test is [failing](https://github.com/openstreetmap/iD/pull/8276/checks?check_run_id=1632225027):

```
      expected 'Missing en translation: inspector.display_name.ref' to deeply equal '66'
4063
```

Not sure why, since I did add the string to both the YAML file and the compiled JSON file.

Reply:
I'd forgotten that the code tests don't actually load the UI strings by default. It couldn't find this string since it didn't have any of the strings.